Apps/PackageTest/0.69.0/android/build.gradle

```diff
@@ -6,8 +6,8 @@ buildscript {
     ext {
         buildToolsVersion = "30.0.2"
         minSdkVersion = 21
-        compileSdkVersion = 31
-        targetSdkVersion = 31
+        compileSdkVersion = 30
```

Comment:
@ryantrem and I discussed this quite a bit and we believe the right thing to do is to always update buildToolsVersion, compileSdkVersion, targetSdkVersion, and gradle to the latest while keeping minSdkVersion as low as possible. All of the versions should be consistent across all of the projects. The NDK should also be updated to the latest, but we can't for now because of the API24 namespace trick.